.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+```
+---
+name: "\U0001F680 Feature request"
```

Comment:
What's the difference between `feature request` & `Enhancement Request`

Reply:
deleted